cli/internal/runsummary/vercel.go

```diff
@@ -49,31 +53,32 @@ type vercelTask struct {
 	Name         string            `json:"name,omitempty"`
 	Workspace    string            `json:"workspace,omitempty"`
 	Hash         string            `json:"hash,omitempty"`
-	StartTime    int               `json:"startTime,omitempty"`
-	EndTime      int               `json:"endTime,omitempty"`
+	StartTime    int64             `json:"startTime,omitempty"`
+	EndTime      int64             `json:"endTime,omitempty"`
 	Cache        vercelCacheStatus `json:"cache,omitempty"`
 	ExitCode     int               `json:"exitCode,omitempty"`
 	Dependencies []string          `json:"dependencies,omitempty"`
 	Dependents   []string          `json:"dependents,omitempty"`
 }
 
-func newVercelRunCreatePayload(runsummary *RunSummary) *vercelRunPayload {
-	startTime := int(runsummary.ExecutionSummary.startedAt.UnixMilli())
-	var context = "LOCAL"
+func (rsm *Meta) newVercelRunCreatePayload() *vercelRunPayload {
```

Comment:
If it's going into the spaces payload, it should be part of the Run Summary. It seems like good information regardless. If you're good with that, we can merge this in and I'll update in my PR

Reply:
Added to execution summary